Packages/Netherlands3D/T3DPipeline/Runtime/Scripts/ParseCityJSON/CityJSONStructure/CityJSON.cs

```diff
@@ -25,7 +25,7 @@ public class CityJSON : MonoBehaviour
         public JSONNode Appearance { get; private set; }
         public JSONNode GeometryTemplates { get; private set; }
 
-        public CityObject[] CityObjects { get; private set; }
+        public CityObject[] CityObjects { get; private set; } = new CityObject[0];
```

Comment:
Zoals net overlegd met Tom. Deze aray wordt geinitialiseerd zodat er geen null exceptions optreden in de foreach loop. Omzetten naar een List betekend dat je ook List functies kan aanroepen, vandaar de keuze voor een property met public lezen, maar niet public schrijven.